pkgs/tools/misc/diskonaut/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "diskonaut";
+  version = "1.15.1";
+
+  src = fetchFromGitHub {
+    owner = "imsnif";
+    repo = "diskonaut";
+    rev = "f716f629c0a2c5e9c20e5e81c6ba260b357c9ed3";
+    sha256 = "0r2f3rljq82c8vzmg9anwxiz3bbsn1n4g9vmnda4hg9kxg96v26a";
+  };
+
+  buildInputs = [ ];
+
+  cargoSha256 = "1gvl78pkjbiy68k4j1g42195skkf3m00pfyhimrac4imrap7nvsw";
```

```suggestion
  cargoSha256 = "03hqdg6pnfxnhwk0xwhwmbrk4dicjpjllbbai56a3391xac5wmi6";
```

Remove empty `buildInputs`, this Cargo SHA-256 corresponds to version 0.3.0.